Packages/com.unity.inputsystem/InputSystem/Plugins/Switch/NPad.cs

```diff
@@ -40,29 +40,31 @@ public FourCC GetFormat()
         [InputControl(name = "leftSR", displayName = "SR (Left)", layout = "Button", bit = (uint)Button.LSR)]
         [InputControl(name = "rightSL", displayName = "SL (Right)", layout = "Button", bit = (uint)Button.RSL)]
         [InputControl(name = "rightSR", displayName = "SR (Right)", layout = "Button", bit = (uint)Button.RSR)]
-        ////TODO: clarify meaning of these buttons
-        [InputControl(name = "leftVK", layout = "Dpad")]
-        [InputControl(name = "leftVK/up", bit = (uint)Button.VKey_LUp)]
-        [InputControl(name = "leftVK/down", bit = (uint)Button.VKey_LDown)]
-        [InputControl(name = "leftVK/left", bit = (uint)Button.VKey_LLeft)]
-        [InputControl(name = "leftVK/right", bit = (uint)Button.VKey_LRight)]
-        [InputControl(name = "rightVK", layout = "Dpad")]
-        [InputControl(name = "rightVK/up", bit = (uint)Button.VKey_RUp)]
-        [InputControl(name = "rightVK/down", bit = (uint)Button.VKey_RDown)]
-        [InputControl(name = "rightVK/left", bit = (uint)Button.VKey_RLeft)]
-        [InputControl(name = "rightVK/right", bit = (uint)Button.VKey_RRight)]
         [FieldOffset(0)]
         public uint buttons;
 
-        ////FIXME: it's so counterintuitive that we have to set "layout" here even though we inherit the control from Gamepad
-        [InputControl(name = "leftStick", layout = "Stick")]
         [FieldOffset(4)]
         public Vector2 leftStick;
 
-        [InputControl(name = "rightStick", layout = "Stick")]
         [FieldOffset(12)]
         public Vector2 rightStick;
 
+        [InputControl(name = "acceleration")]
```

Comment:
If the name matches the field name, no need to specify it on the attribute.

Reply:
Interesting. That sounds like a bug. I'll give it a try after this PR is merged.